tests/test_upload_manager.py

```diff
@@ -32,5 +37,56 @@ def test_upload_manager_switches_drivers():
     container.bind('StorageConfig', storage)
     container.bind('UploadDiskDriver', object)
     container.bind('UploadTestDriver', object)
-
     container.bind('UploadManager', UploadManager(container).driver('test'))
+
+
+def test_upload_manage_accept_files():
+    container = App()
+
+    container.bind('Application', application)
+    container.bind('StorageConfig', storage)
+    container.bind('UploadDiskDriver', UploadDiskDriver)
+
+    class ImageTest():
+        """
+        Image test for emulate upload file
+        """
+
+        @property
+        def filename(self):
+            return os.getcwd() + "/tests/static/test.jpg"
+
+        @property
+        def file(self):
+            return open(self.filename, 'rb')
+
+    img = ImageTest()
+    UploadManager(container).driver('disk').accept('jpg', 'png').store(img)
+
+
+def test_upload_manage_accept_files_error():
+    container = App()
+
+    container.bind('Application', application)
+    container.bind('StorageConfig', storage)
+    container.bind('UploadDiskDriver', UploadDiskDriver)
+
+    class ImageTest():
```

Comment:
Lets move this image class outside of the test functions. This way we only need 1 class for all tests instead of copying and pasting the same class. Make it DRY

Reply:
Yes, it's true. I did not realize